docs/metrics/difficulty-score.md

```diff
@@ -0,0 +1,215 @@
+---
+description: How to find common model regressions and failures
+---
+
+# Difficulty Score
+
+Difficulty scores are automatically computed within Kolena to surface datapoints that commonly contribute to poor
+model performance. Difficulty scores consider a user's custom
+[Quality Standard](../dataset/core-concepts/index.md#quality-standard) configuration to make
+an informed assessment of which datapoints lead to the greatest recurring problems across all models using multiple
+performance indicators. Difficulty scores range from 0 to 1, where a lower difficulty score
+indicates that models produce the ideal datapoint-level metrics (e.g. lower inference time, higher accuracy),
+and a higher difficulty score indicates that models consistently face problems or "difficulty"
+(e.g. longer inference time, lower BLEU scores, and/or lower recall).
+
+When one model is selected in Studio, the difficulty score of a datapoint is the difficulty score derived
+from that model's results. When more than one model is considered, the overall difficulty score of a datapoint is
+the average value from each difficulty score from each model's results.
+
+??? "Using Difficulty Scores for Regression Testing"
+    With two models called `old` and `new` are selected in Studio so users see two model-level difficulty scores,
+    and one overall difficulty score for any datapoint. With a filter for
+    `new.difficulty_score > old.difficulty_score`, we find all the datapoints that performed worse in the
+    `new` model, which highlights the regressions.
+
+    With a filter for `datapoint.difficulty_score > 0.9`, we see all the datapoints that significantly struggle
+    across both the `old` and `new` models, which are common failures that persist over different model iterations.
+
+## Implementation Details
```

Comment:
Before we jump to the implementation details, should we summarize the logic here? Also, can this be summarized into a formula? One formula for model-level difficulty score and one for the aggregate difficulty score.